valohai_yaml/pipelines/conversion.py

```diff
@@ -26,10 +39,21 @@ def __init__(
         self.config = config
         self.commit_identifier = commit_identifier
 
-    def convert_pipeline(self, pipeline: Pipeline) -> Dict[str, List[ConvertedObject]]:
+    def convert_pipeline(self, pipeline: Pipeline) -> ConvertedPipeline:
         return {
             "edges": [edge.get_expanded() for edge in pipeline.edges],
             "nodes": [self.convert_node(node) for node in pipeline.nodes],
+            "parameters": {
+                parameter.name: self.convert_parameter(parameter)
+                for parameter in pipeline.parameters
+            },
+        }
+
+    def convert_parameter(self, parameter: PipelineParameter) -> ConvertedObject:
+        """Convert a pipeline parameter to a config-expression payload."""
+        return {
+            "config": {**parameter.serialize()},
+            "expression": parameter.default,
```

Comment:
Lets mimic the way the web UI works and put empty string here when the default is missing.